tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -129,9 +129,9 @@ Status ModularFileSystem::FileExists(
   return StatusFromTF_Status(plugin_status.get());
 }
 
-bool ModularFileSystem::FilesExist(
-    const std::vector<std::string>& files,
-    std::vector<Status>* status /*, TransactionToken* token */) {
+bool ModularFileSystem::FilesExist(const std::vector<std::string>& files,
+                                   std::vector<Status>* status,
+                                   TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `status` which will hold the results